test/test.oauth2.ts

```diff
@@ -642,7 +642,9 @@ it('should pass due to valid issuer', async () => {
 it('should be able to retrieve a list of Google certificates', (done) => {
   const scope = nock('https://www.googleapis.com')
                     .get(certsPath)
-                    .replyWithFile(200, certsResPath);
+                    .replyWithFile(200, certsResPath, {
+                      'Content-Type': 'application/json',
```

Comment:
There is a fix here for Gaxios that will fix this issue:
https://github.com/JustinBeckwith/gaxios/pull/21

Need to wait for this land before we release it:
https://github.com/googleapis/gcp-metadata/pull/152

Reply:
Got rid of it!